src/resources/postcss/base/forms/_text.pcss

```diff
@@ -0,0 +1,33 @@
+.tribe-events {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * Form Control: Text
+	 *
+	 * Example:
+	 * <div class="tribe-common-form-control-text">
+	 *   <label for="textInput">Text Input</label>
+	 *   <input id="textInput" name="textInput" type="text" placeholder="Text Input" />
+	 * </div>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-common-form-control-text {
```

Comment:
Curious, is this a special form input styling for text inputs only or should this be a bit more global in nature for use on fields that are more generic and will have this style?

Reply:
Not used for radios, checkboxes, sliders, or toggles. More for inputs (text, number, email, etc). In the style guide we didn't have any other input types, so I went with text. I can change it to `-input` instead.

Reply:
Cool that makes sense, but yeah a more generic class name would be helpful 👍